spec/factories/harvests.rb

```diff
@@ -6,7 +6,7 @@
     plant_part
     planting nil
     owner { planting.present? ? planting.owner : FactoryBot.create(:member) }
-    harvested_at Time.zone.local(2015, 9, 17)
+    harvested_at { Time.zone.local(2015, 9, 17) }
```

Comment:
Thats... not really dynamic!

Reply:
true! I think there's some tests that should be fixed to use TimeCop